### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by MattGomb**
In this PR I created the finalized styling for the project.

- I adjusted some colors, some font-sizes, and properly placed everything
- I created the mobile-first approach, everything scales with size

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -14,8 +14,10 @@ body {
 
 .nav-container {
```
Comment (danmainah) [inline src/index.css:15]: - [x] @MattGomb great work on your design, the color and alignment should match, however, some elements don't match the design given by zeplin, below is a comparison with your design, I have highlighted the sections for easier identification:

Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @MattGomb , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.css (hunk 1 above)
Comment (MattGomb) [reply src/index.css:15]: Hi @danmainah, thank you for the observations! Would it be possible for me to leave the title colored like that..? I did it on purpose, to have a little something that stands out. 

I fixed the alignment of the form-title.

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @MattGomb 

Your project is complete! Great job for making the requested changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- well styled
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

- [optional sugestion] excellent work. it is always preferable to follow the design of the project, you can always edit it to what you want later on. 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**